controller/metrics/metrics_test.go

```diff
@@ -40,7 +44,7 @@ metadata:
 spec:
   destination:
     namespace: dummy-namespace
-    server: https://localhost:6443
+    name: cluster1
```

Comment:
I switched the test apps to destination name to force the logic that calculates dest_server based on name.